project.md

```diff
@@ -58,11 +179,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think you'd be able to engage with the language quite frequently because the
+whole point is to create some way of transcribing taiko from something that is
+only heard to something that can be written down. There are already conventions
```

Comment:
Even if you use something like MuseScore as a model, I think there are enough differences between taiko and piano for there to be enough language design decisions for this project!